### DISCUSSION TITLE
Keycloak can't access certificates in 23.0.3

### DISCUSSION THREAD

**Discussion by zd14a** [keycloak/keycloak#25722] [Q&A] [answered]
Hi,

I tried to update my running Keycloak instance to the most current 23.0.3 version. Unfortunately this didn't work, because Keycloak wasn't able to access the certificates in the conf directory. In addition I set up keycloak on a new system and got the exact same issue.

I have a Windows 2022 System and in the keycloak.conf the following settings  

```
...
https-certificate-file=${kc.home.dir}conf/server01.2023.crt
https-certificate-key-file=${kc.home.dir}conf/server01.2023.key
...
```

When I try to start Keycloak I get the following error (in both systems)

> ERROR [org.keycloak.quarkus.runtime.cli.ExecutionExceptionHandler] (main) ERROR: C:Program FilesKeycloakkeycloak-23.0.3bin..conf\server01.2023.crt

As you can see the path where it try to access the certificate does not contain any back slashes. But it converts the (normal) slash into a back slash. I already tried to replace the variable with the full path, but the result is the same, as I need to use the backslash for, at least, the volume. Seems to me like an error in handling the paths and slashes. 

Does anybody see the same problems in their environment? Any idea for a solution?

**Comment by shawkins** [comment, marked as answer]
Duplicate of #23217

@Pepo48 @vmuzikar should a backport to 23 be considered?

**Comment by zd14a** [comment]
@shawkins 
Sorry, thought I searched carefully, but for some reason I didn't find this issue. It seems to be the same.

**Reply by shawkins** [reply]
No problem
